cis/system/ksp-audit-cis-mysql-2-1-7.yaml

```diff
@@ -0,0 +1,22 @@
+# KubeArmor is an open source software that enables you to protect your cloud workload at run-time.
+# To learn more about KubeArmor visit:
+# https://www.accuknox.com/kubearmor/
+
+apiVersion: security.kubearmor.com/v1
+kind: KubeArmorPolicy
+metadata:
+  name: ksp-audit-cis-mysql-2-17
+  namespace: default
+spec:
+  tags: ["MySQL", "CIS"]
```

Comment:
please add the CIS rule number or description to the tags @yasin-cs-ko-ak

Reply:
added